example_configs/weblogic.yml

```diff
@@ -0,0 +1,60 @@
+---
+jmxUrl: service:jmx:iiop:///jndi/iiop://localhost:7001/weblogic.management.mbeanservers.runtime
+username: weblogic
+password: XXXXXXX
+lowercaseOutputName: true
+lowercaseOutputLabelNames: true
+whitelistObjectNames:
+  - "com.bea:Name=*,Type=ServerRuntime"
+  - "com.bea:ServerRuntime=*,Type=ApplicationRuntime,*"
+  - "com.bea:ServerRuntime=*,Type=JDBCDataSourceRuntime,*"
+  - "com.bea:ServerRuntime=*,Type=JMSDestinationRuntime,*"
+  - "com.bea:ServerRuntime=*,Type=SAFRemoteEndpointRuntime,*"
+  - "com.bea:ServerRuntime=*,Type=ThreadPoolRuntime,*"
+
+rules:
+  - pattern: "^com.bea<ServerRuntime=.+, Name=(.+), Type=JDBCDataSourceRuntime><>(.+): (Running|Suspended|Shutdown|Overloaded|Unknown)$"
```

Comment:
As this is doing something a bit unusual, it'd be clearer to change the 2nd capture group to `State`